okhttp/src/test/java/okhttp3/CallTest.java

```diff
@@ -889,15 +890,15 @@ private void postBodyRetransmittedAfterAuthorizationFail(String body) throws Exc
     server.enqueue(new MockResponse().setBody("abc"));
     server.enqueue(new MockResponse().setBody("def").throttleBody(1, 750, TimeUnit.MILLISECONDS));
 
-    // First request: time out after 1000ms.
+    // First request: time out after 1s.
     client = client.newBuilder()
-        .readTimeout(1000, TimeUnit.MILLISECONDS)
+        .readTimeout(Duration.ofSeconds(1))
```

Comment:
Changed the time unit as I got another warning when I had tried with `Duration.ofMillis(1000)`
```
okhttp/okhttp/src/test/java/okhttp3/CallTest.java:895: warning: [CanonicalDuration] Duration can be expressed more clearly with different units
        .readTimeout(Duration.ofMillis(1000))
                                      ^
    (see https://errorprone.info/bugpattern/CanonicalDuration)
  Did you mean '.readTimeout(Duration.ofSeconds(1))'?
```